src/tools/pub/dependencies.md

```diff
@@ -441,21 +441,34 @@ Use [caret syntax](#caret-syntax)
   but where a new version `2.x` would be a major upgrade
   that isn't semantically compatible with `1.x` versions. 
 
-Depend on the latest stable package versions
-: Use [`pub upgrade`][] to update to the latest package versions
+### Depend on the latest stable package versions
+ Use [`pub upgrade`][] to update to the latest package versions
   that your pubspec allows.
   To identify dependencies in your app or package that
   aren't on the latest stable versions,
   use [`pub outdated`][].
 
-Test whenever you update package dependencies
-: If you run [`pub upgrade`][] without updating your pubspec,
+### Test whenever you update package dependencies
+ If you run [`pub upgrade`][] without updating your pubspec,
```

Comment:
Should this be `dart pub upgrade` or `flutter pub upgrade`? The standalone `pub` command is no more...

Reply:
Yeah we tend to prefer just `dart pub ...` on the Dart site now, especially if linking to the command.